kratos/python_scripts/print_info_in_file_process.py

```diff
@@ -0,0 +1,118 @@
+            "integration_point_number" : 0,
+            "erase_previous_info"      : true,
+            "output_interval"          : 1
+        }
+        """
+        )
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable_name"].GetString())
+        self.is_nodal_variable_type = settings["variable_type"].GetString() == "nodal"
+        self.file_name = settings["file_name"].GetString()
+        self.output_control_type = settings["output_control_type"].GetString()
+        self.output_interval = settings["output_interval"].GetDouble()
+        self.model_part = Model[settings["model_part_name"].GetString()]
+        self.instant_previous_plot = 0.0
+        self.integration_point = settings["integration_point_number"].GetInt()
+
+        open_file_aproach = "a"
+        if settings["erase_previous_info"].GetBool():
+            open_file_aproach = "w"
+
+        self.plot_file = open(self.file_name, open_file_aproach)
+        self.plot_file.write("In this file we print the " + settings["variable_type"].GetString() + " " + settings["variable_name"].GetString() + " in the sub_model_part: " + settings["model_part_name"].GetString() + "\n\n")
+        self.plot_file.write(" TIME      \t" + settings["variable_name"].GetString() + "\n")
+        self.plot_file.close()
+
+
+    def PrintOutput(self):
+        self.SetPreviousPlotInstant()
+        if self.is_nodal_variable_type: # In nodal values we add them
+            for node in self.model_part.Nodes:
+                counter = 0
+                if counter == 0:
+                    array_values = self.GetValueToPrint(node)
+                if counter > 0:
+                    for value in array_values:
+                        value += self.GetValueToPrint(node)[counter]
+                counter += 1
+            self.plot_file = open(self.file_name, "a")
+            self.plot_file.write("{0:.4e}".format(self.__GetTime()).rjust(11) + "\t")
+            for value in array_values:
+                self.plot_file.write("{0:.4e}".format(value).rjust(11) + "\t")
+            self.plot_file.write("\n")
+            self.plot_file.close()
+        else: # elemental information, not adding values
+            for elem in self.model_part.Elements:
+                counter = 0
+                if counter == 0:
+                    array_values = self.GetValueToPrint(elem)
+                for value in array_values:
+                    if counter > 0:
+                        value += self.GetValueToPrint(elem)[counter]
+                    counter += 1
+            self.plot_file = open(self.file_name, "a")
+            self.plot_file.write("{0:.4e}".format(self.__GetTime()).rjust(11) + "\t")
+            for value in array_values:
+                self.plot_file.write("{0:.4e}".format(value).rjust(11) + "\t")
+            self.plot_file.write("\n")
+            self.plot_file.close()
```

Comment:
TBH I don't understand the logic

why do you add nodal but not elemental values?
I think this should be an option
How do you use this process e.g. for `DISPLACEMENT` with more than one node?

Reply:
Elemental values have in general more than one IP, if we add the elements maybe we should add also the IP values.
This process works with SubModels so if this submodel has more than one node it will add automatically their values of the Displacement.

Reply:
hm ok but then I would rename the process, the name doesn't indicate what it is doing
and IMO adding displacements doesn't make sense. Reactions sure. Also it doesn't make sense for IPs no?
How about throwing when there is more than one entity in the ModelPart?

Reply:
It is complex in the sense that depending on the application, one implementation or another could be useful... In my personal case, I would like to track the `DISPLACEMENT `of a node (one block in the .json) and another file printing the `REACTION `in the whole model_part. Another case is tracking the STRAIN or STRESS in a certain Element and IP (without adding any elements/IP).

Reply:
I understand the usecase, but I think this is very dangerous to use wrong
if you accidentally use a ModelPart with more than one node then it no longer makes sense in some cases

=> i.e. I would add a setting `sum_results_from_multiple_entites` which is `false` by default (I couldn't come up with a better name :sweat_smile:)
this way the you can do what you want to do while at the same time have some protection from wrong input

Reply:
good for me, currently on it

Reply:
the you could also sum the results from IP vars (I could imagine e.g. sth like the `FLOW_RATE` in case of fluids)

Reply:
done now like that